net/modemmanager/README.md

```diff
@@ -38,3 +39,6 @@ It will default to 'ipv4' if not given.
 
 The 'signalrate' option set's the signal refresh rate (in seconds) for the device.
 You can call signal info with command: mmcli -m 0 --signal-get
+
+The 'rat' option set's the Radio access technology for the device.
+You can get the RAT info with command: mmcli -m 0 or with AT+QENG="servingcell AT command.
```

Comment:
AT+QENG is Quectel specific, I would document here that `mmcli -m 0` reports the list of allowed/preferred modes the specific modem supports, and so the given settings should be one of the combinations given in the list.